linera-execution/src/transaction_tracker.rs

```diff
@@ -34,11 +37,17 @@ pub struct TransactionTracker {
     /// Events recorded by contracts' `emit` calls.
     events: Vec<Event>,
     /// Blobs created by contracts.
+    ///
+    /// As of right now, blobs created by the contracts are one of the two:
+    /// - OpenChain
+    /// - CreateApplication
     blobs: BTreeMap<BlobId, Blob>,
     /// Operation result.
     operation_result: Option<Vec<u8>>,
     /// Streams that have been updated but not yet processed during this transaction.
     streams_to_process: BTreeMap<ApplicationId, AppStreamUpdates>,
+    /// Published blobs this transaction refers to by BlobId.
```

Comment:
```suggestion
    /// Published blobs this transaction refers to by [`BlobId`].
```

Reply:
Done in [Fix typos](https://github.com/linera-io/linera-protocol/pull/3995/commits/c9563c96589bf5511e29f652dcc6ebfddcf5bbb6).